libs/beacon/tests/unit/dkg_resilience.cpp

```diff
@@ -639,7 +639,7 @@ void GenerateTest(uint32_t cabinet_size, uint32_t threshold, uint32_t qual_size,
   }
 }
 
-TEST(dkg_setup, bad_messages)
+TEST(dkg_setup, DISABLED_bad_messages)
 {
```

Comment:
Is there a reason these are being disabled again?

Reply:
Don't know. Let's not disable them.

Reply:
Un-disabled them. I think it was due to them taking so long they occasionally time out